app/controllers/userController.js

```diff
@@ -100,15 +103,14 @@ let userControl = {
       .then((document) => {
         result = document.filter((doc) => {
           if (doc.public) {
-            console.log('public');
             return doc;
```

Comment:
**Nitpick - Code quality**
You should be able to pull the user documents in one round trip to the database. Maximize relationships in your models and you'd be fine. `findAll` returns an array and that's enough for your client. Make your API learn and let it do its work. Your client expects an array and I think some of the checks are not necessary